accounts/views.py

```diff
@@ -49,14 +77,42 @@ def post(self, request, *args, **kwargs):
     description="User Sign-In (Login) Endpoint",
     tags=["Users"],
 )
-class UserLoginAPIView(generics.CreateAPIView):
+class UserLoginAPIView(APIView):
     serializer_class = UserLoginSerializer
 
-    def create(self, request, *args, **kwargs):
-        serializer = self.get_serializer(data=request.data)
-        if serializer.is_valid(raise_exception=True):
-            return Response(serializer.validated_data, status=status.HTTP_200_OK)
+    def post(self, request, *args, **kwargs):
+        serializer = self.serializer_class(data=request.data)
+
+        try:
+            serializer.is_valid(raise_exception=True)
```

Comment:
Please Up to now It seems it can check the username and password convention is not checking the convention of the username, for instance I used ``` username = ee and password = ee ``` And it is returning the ``` invalid credential ``` error which means it is taking this to database while it is fake username even password which is not following the convention.

Reply:
@AbuuIbtisaam @usanaphtal112 could you meet and resolve this each other like they're taking time here.

Reply:
@leandreAlly I wanted a suggestion on error message and status code but on doing some research I found 422 - Unprocessable Content. 
<img width="472" alt="image" src="https://github.com/SahanaTeam/sahaget-bn/assets/103914775/a6c8e0b0-b1d7-42dc-a705-03e3ef052485">

Reply:
@usanaphtal112 Now you can test it